java/arcs/android/crdt/ParcelableVersionMap.kt

```diff
@@ -26,4 +25,4 @@ fun VersionMap.toProto(): VersionMapProto = VersionMapProto.newBuilder()
 
 /** Reads a [VersionMap] out of a [Parcel]. */
 fun Parcel.readVersionMap(): VersionMap? =
-    readProto(VersionMapProto.getDefaultInstance())?.let { VersionMap(it) }
+    readProto(VersionMapProto.getDefaultInstance())?.let { fromProto(it) }
```

Comment:
Should this file be renamed to `ProtoVersionMap.kt`?

Reply:
Done (went with `VersionMapProto.kt` -- named the same as the proto itself)